tools/stronghold/src/api/checker.py

```diff
@@ -16,12 +17,36 @@ def run() -> None:
     args = parser.parse_args(sys.argv[1:])
 
     repo = api.git.Repository(pathlib.Path('.'))
+
+    # By default, our GitHub jobs only fetch to a depth of one. This
+    # means that the base commit will not be known to our local
+    # clone. We must fetch it in order to compare head and base.
+    #
+    # The fetch is a smidge noisy, hide it by default.
+    print('::group::fetch github.event.pull_request.base.sha')
+    repo.run(['fetch', 'origin', args.base_commit], check=True)
+    print('::endgroup::')
+
     violations = api.compatibility.check_range(
         repo, head=args.head_commit, base=args.base_commit
     )
     if len(violations) == 0:
         return
+
+    pinfo = repo.run(
+        [
+            'show',
+            # Don't show the file contents.
+            '--no-patch',
+            # Show the title and the full commit message.
+            '--pretty=format:%B',
+        ],
+        check=True,
+        stdout=subprocess.PIPE,
+    )
+    suppressed = '#suppress-api-compatibility-check' in pinfo.stdout
```

Comment:
Just to clarify, the user needs to add `#suppress-api-compatibility-check` in their commit message (or title) if they want to suppress stronghold checks. If suppressed, it will still log them in the job output but not fail the job.

Reply:
correct! and they will still show up in the files view.

what do you think?